plugins/org.eclipse.fordiac.ide.typemanagement/src/org/eclipse/fordiac/ide/typemanagement/refactoring/UpdateTypeLibraryEntryChange.java

```diff
@@ -35,14 +35,14 @@
 import org.eclipse.ui.IEditorPart;
 import org.eclipse.ui.part.FileEditorInput;
 
-public class UpdateTypeEntryChange extends Change {
+public class UpdateTypeLibraryEntryChange extends Change {
```

Comment:
Please dont rename that. It is called TypeEntry through the whole code base, so we want to stay consistent in naming, no matter of personal taste